src/components/IPScan/Summary/serializers.js

```diff
@@ -12,6 +12,9 @@ type Location = {
 }
 */
 
+const OTHER_FEATURES_DBS = ['funfam'];
```

Comment:
Does InterProScan reports Pfam-N matches? If so I think Pfam-N should also be added to that list.

Reply:
Correct. Pfam-N matches were provided by Google Research and are not reported by InterProScan.